api/src/pcapi/routes/backoffice/operations/blueprint.py

```diff
@@ -110,3 +114,109 @@ def create_event() -> utils.BackofficeResponse:
         )
 
     return redirect(url_for(".list_events"), code=303)
+
+
+@operations_blueprint.route("/<int:special_event_id>", methods=["GET"])
+@atomic()
+def get_event_details(special_event_id: int) -> utils.BackofficeResponse:
+    special_event_query = operations_models.SpecialEvent.query.filter(
+        operations_models.SpecialEvent.id == special_event_id
+    ).options(
+        sa.orm.joinedload(operations_models.SpecialEvent.questions).load_only(
+            operations_models.SpecialEventQuestion.id,
+            operations_models.SpecialEventQuestion.externalId,
+            operations_models.SpecialEventQuestion.title,
+        ),
+        sa.orm.joinedload(operations_models.SpecialEvent.responses)
+        .load_only(
+            operations_models.SpecialEventResponse.id,
+            operations_models.SpecialEventResponse.dateSubmitted,
+            operations_models.SpecialEventResponse.email,
+            operations_models.SpecialEventResponse.phoneNumber,
+            operations_models.SpecialEventResponse.status,
+            operations_models.SpecialEventResponse.userId,
+        )
+        .joinedload(operations_models.SpecialEventResponse.answers)
+        .load_only(
+            operations_models.SpecialEventAnswer.id,
+            operations_models.SpecialEventAnswer.questionId,
+            operations_models.SpecialEventAnswer.text,
+        ),
+        sa.orm.joinedload(operations_models.SpecialEvent.responses)
+        .joinedload(operations_models.SpecialEventResponse.user)
+        .load_only(
+            users_models.User.id,
+            users_models.User.firstName,
+            users_models.User.lastName,
+            users_models.User.phoneNumber,
+            users_models.User.email,
+            users_models.User.roles,
+        ),
+    )
+    special_event = special_event_query.one_or_none()
+
+    if not special_event:
+        raise NotFound()
+
+    return render_template(
+        "operations/details.html",
+        special_event=special_event,
+        active_tab=request.args.get("active_tab", "responses"),
+    )
+
+
+@operations_blueprint.route("/<int:special_event_id>/validate/<int:response_id>", methods=["POST"])
```

Comment:
On peut demander plusieurs avis, mais j'aurais mis d'abord l'id de la candidature, puis l'action, par logique dans le chemin. Idem en-dessous.
```suggestion
@operations_blueprint.route("/<int:special_event_id>/responses/<int:response_id>/validate", methods=["POST"])
```